crates/lib/src/client.rs

```diff
@@ -115,6 +113,16 @@ impl Qcs {
     }
 }
 
+impl Default for Qcs {
+    fn default() -> Self {
+        Self::with_config(
+            ClientConfiguration::builder()
+                .build()
+                .expect("builder should be valid with all defaults"),
```

Comment:
This is necessary because `ClientConfiguration::default()` was removed, see https://github.com/rigetti/qcs-api-client-rust/commit/667d36982d68908182fae5b9a171e344fe26c4d1#diff-011700f7dbafb08434174266533fb9f2840cb110e45a37a9cacded854093869cL404